sormas-app/app/src/main/java/de/symeda/sormas/app/backend/immunization/Immunization.java

```diff
@@ -293,4 +305,30 @@ public void setCountry(Country country) {
     public String getI18nPrefix() {
         return I18N_PREFIX;
     }
+
+
+    public void update(Immunization immunization) {
```

Comment:
What is this method used for?

Reply:
to update the immunization in case the user chooses to override the existing data

Reply:
Please check whether you can use AbstractAdoDao.mergeOrCreate instead because this method would have to be maintained, which will almost certainly lead to problems later; if that's not possible at all, please add the new fields here that are missing (e.g. facility fields)